docker/centos7-builder.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:7 AS build-stage
```

Comment:
fyi: For the `docker buildx` follow-up: unfortunately there is no `centos:7` for `s390x`. There is a `clefos:8` or centos stream 9.